contracts/LendingPool.sol

```diff
@@ -35,9 +35,9 @@ contract LendingPool is OwnableUpgradeable, ERC721Upgradeable, Clone {
     uint public totalBorrowed; // = 0;
     string private constant baseURI = "https://nft.llamalend.com/nft/";
     uint maxDailyBorrows; // IMPORTANT: an attacker can borrow up to 150% of this limit if they prepare beforehand
-    uint216 currentDailyBorrows;
+    uint216 currentDailyBorrows; // would have to borrow more eth than will ever exist daily to break
     uint40 lastUpdateDailyBorrows;
-    mapping(address => bool) public liquidators;
+    mapping(address => uint) public liquidators;
```

Comment:
have you verified that this is in fact cheaper? i  think it should be but havent verified it

Reply:
Saves 16168 gas on deployment and a whopping 12 gas on addLiquidator()